sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -44,31 +45,10 @@ export interface AnalyzeResult {
 export type AnalyzeTextOptions = OperationOptions & AnalyzeRequest;
 
 // @public
-export interface AnswerResult {
```

Comment:
Did these get moved or no longer needed? I guess why are they missing and is that expected?

Reply:
They were renamed to `QueryAnswer` and `QueryCaption`.